src/Input/input.stories.mdx

```diff
@@ -75,3 +75,15 @@ The dark version of the input is meant to be used against a `deep-purple` backgr
     </div>
   </Story>
 </Preview>
+
+# With Addon
```

Comment:
Should we call this prefix and suffix instead?

Reply:
Done. Thanks!